node/forcedroid.js

```diff
@@ -425,9 +424,6 @@ function createArgsProcessorList() {
     // Target dir
     addProcessorFor(argProcessorList, 'targetdir', 'Enter the target directory of your app:', 'Invalid value for target dir: \'$val\'.',  /^\S+$/);
 
-    // Target API 
```

Comment:
No reason to get ```targetApi``` from the user, especially since it's used only for native apps. We use the one from the ```Mobile SDK```.